webapp/repositories.py

```diff
@@ -74,6 +74,14 @@ def get_by_id(self, group_id: int) -> Group:
             group = session.query(Group).get(group_id)
             return group
 
+
+    def get_group_id_by_name(self, group_name: str) -> int:
+        groups = self.get_all()
+        for group in groups:
+            if group_name == group.__getattribute__("title"):
```

Comment:
Есть подозрение, что можно сделать:
```suggestion
            if group_name == group.title:
```
Аналогично со строчкой ниже: `return group.id`. См. https://docs.sqlalchemy.org/en/14/orm/tutorial.html

Reply:
Исправил